examples/experimental/dagster-airlift/README.md

```diff
@@ -1,26 +1,174 @@
+
+For most projects we anticipate the following process:
+
+- Create a new factory class for every operator type or specialized use case of an operator (e.g. a BashOperator invoking a dbt project).
+- Use that factory to create definitions associating with each Airflow task.
+
+In our example we have a pre-written factory class (in `examples/experimental/dagster-airlift/dagster_airlift/dbt/multi_asset.py`) that is associated with invoking a dbt project:
+
+```python
+@dataclass
+class DbtProjectDefs(DefsFactory):
+    dbt_project_path: Path
+    name: str
+    group: Optional[str] = None
+
+    def build_defs(self) -> Definitions:
+        dbt_manifest_path = self.dbt_project_path / "target" / "manifest.json"
+
+        @dbt_assets(manifest=json.loads(dbt_manifest_path.read_text()), name=self.name)
+        def _dbt_asset(context: AssetExecutionContext, dbt: DbtCliResource):
+            yield from dbt.cli(["build"], context=context).stream()
+
+        if self.group:
+            _dbt_asset = _dbt_asset.with_attributes(
+                group_names_by_key={key: self.group for key in _dbt_asset.keys}
+            )
+
+        return Definitions(
+            assets=[_dbt_asset],
+            resources={
+                "dbt": DbtCliResource(
+                    project_dir=self.dbt_project_path, profiles_dir=self.dbt_project_path
+                )
+            },
+        )
+```
+
+We imagine most people will have to customize this, so we encourage you to copy and paste it for now.
+
+Then you can add instances of the Factory to your `build_defs_from_airflow_instance` call:
+
+```python
+
+defs = build_defs_from_airflow_instance(
+    airflow_instance=airflow_instance,
+    orchestrated_defs=defs_from_factories(
+        DbtProjectDefs(
+            name="dbt_dag__build_dbt_models",
+            dbt_project_path=dbt_project_path(),
+            group="dbt",
+        ),
+    ),
+)
+```
+
+### Mapping assets to tasks
+
+We default to a "convention over configuration" approach to affiliate Dagster assets with Airflow tasks.
+
+Note the naming convention `dbt_dag__build_dbt_models`. By default we use this convention to encode the dag name (`dbt_dag`) and task id (`build_dbt_models`) of the Airflow task actually orchestrating the computation.
+
+This name gets parsed and set corresponding tags (`airlift/dag_id` and `airlift/task_id`). Alternatively you can set those tags explicitly if you don’t want to rely on the naming convention.
+
+Once this is done you should be able to reload your definitions and the see the full dbt project. Once you run the corresponding Airflow Dag and task, each task completion will corresponding to an asset materialization in any asset that is orchestrated by that task.
+
+_Note: There will be some delay as this process is managed by a Dagster sensor that polls the Airflow instance for task history. This is every 30 seconds by default, so there will be some delay._
```

Comment:
I've actually made the default sensor polling interval very aggressive for the demos. Currently like 1 second.
But likely still will be some delay.